examples/stm32c0x/blink-clock/main.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2024, Jörg Ebeling
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <modm/platform.hpp>
+
+#include "../custom-board.hpp"
+
+#define BLINK_MILLIS 1000
+
+using namespace modm::platform;
+using namespace std::chrono_literals;
+
+uint32_t next_blink = modm::Clock::now().time_since_epoch().count() + BLINK_MILLIS;
+
+int
+main()
+{
+	Board::initialize();
+	Board::LedGn::set();
+	while (true)
+	{
+		if (modm::Clock::now().time_since_epoch().count() >= next_blink)
+		{
+			next_blink += BLINK_MILLIS;
+			Board::LedGn::toggle();
+			Board::LedRd::toggle();
+		}
+	}
+}
```

Comment:
This can be done in a cleaner way without falling back to raw ints instead of strongly-typed std::chrono types:

```c++
using namespace modm::platform;
using namespace std::chrono_literals;

int
main()
{
	Board::initialize();
	Board::LedGn::set();
	
	constexpr auto blick_interval = 1000ms;
	auto next_blink = modm::Clock::now();

	while (true)
	{
		if (modm::Clock::now() >= next_blink)
		{
			next_blink += blick_interval;
			Board::LedGn::toggle();
			Board::LedRd::toggle();
		}
	}
}
```

But this still has a bug on timestamp overflow. Shouldn't we rather show `modm::PeriodicTimer` in a usage example?

Reply:
Ah, sorry I meant to remove these examples, hence the WIP. A custom board is not maintainable for us.